File: beginner/javascript/1184.js
let input = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n'); 

let operation = input.shift();
let cont = 0, i, j;

let mat = new Array();

for(i = 0; i < 12; i++){
    mat.push(new Array());
    for(j = 0; j < 12; j++){
        mat[i][j] = parseFloat(input.shift());
    }
}

const soma_or_media = (mat,op) => {
    let sum = 0;
    for(i = 0; i < 12; i++){
        for(j = 0; j < 12; j++){
            sum += mat[i][j];
            cont++;
        }
    }

    if(op === 'S')
        return sum;
    else if(op === 'M')
        return sum / cont;
};

console.log(soma_or_media(mat,operation).toFixed(1));

File: beginner/python/1184.py
op = input()

mat = [[0 for j in range(12)] for i in range(12)]

for i in range(0,12):
    for j in range(0,12):
        mat[i][j] = float(input())
        
def soma_or_media(mat, op):
    sum = 0
    cont = 0
    for i in range(0,12):
        for j in range(0,12):
            if i > j:
                sum += mat[i][j]
                cont+=1
    
    if op == 'S':
        return sum
    else:
        return sum / cont

print("%.1f" % (soma_or_media(mat,op)))